<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Busqueda</class>
 <widget class="QDialog" name="Dialog_Busqueda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Búsqueda</string>
  </property>
  <widget class="QComboBox" name="CB_TipoBusqueda">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Coincidencia exacta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Subcadena</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de búsqueda</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_TipoBusqueda_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Conector</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>551</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLineEdit" name="LE_CodPostal"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="LE_Telefono"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Correo electrónico</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LE_Direccion"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LE_Nombre"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Teléfono</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="LE_Email"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Código postal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LE_Poblacion"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Dirección</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Población</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_Buscar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
